src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationExtensions.cs

```diff
@@ -64,7 +71,52 @@ public static IConfigurationBuilder AddAzureAppConfiguration(
+        /// </summary>
+        /// <param name="configurationBuilder">The configuration builder to add key-values to.</param>
+        /// <param name="endpoint">The endpoint used to connect to the configuration store.</param>
+        /// <param name="credential">The token credential used to authenticate requests to the configuration store.</param>
+        /// <param name="optional">Determines the behavior of the App Configuration provider when an exception occurs while loading data from server. If false, the exception is thrown. If true, the exception is suppressed and no settings are populated from Azure App Configuration.
+        /// <exception cref="ArgumentException"/> will always be thrown when the caller gives an invalid input configuration (connection strings, endpoints, key/label filters...etc).
+        /// </param>
+        /// <returns>The provided configuration builder.</returns>
+        public static IConfigurationBuilder AddAzureAppConfiguration(
+            this IConfigurationBuilder configurationBuilder,
+            Uri endpoint,
+            TokenCredential credential,
+            bool optional = false)
+        {
+            return configurationBuilder.AddAzureAppConfiguration(options => options.Connect(endpoint, credential), optional);
+        }
+
+        /// <summary>
+        /// Adds key-value data from a primary Azure App Configuration store and one or more replica stores to a configuration builder using endpoints with AAD authentication.
+        /// This is a simplified overload that loads all key-values with no label. For advanced scenarios such as selecting specific keys, 
+        /// filtering by labels, configuring refresh, using feature flags, or resolving Key Vault references, 
+        /// use the overload that accepts an <see cref="Action{AzureAppConfigurationOptions}"/> parameter with options.Connect().
+        /// </summary>
+        /// <param name="configurationBuilder">The configuration builder to add key-values to.</param>
+        /// <param name="endpoints">The list of endpoints used to connect to the configuration store and its replicas.</param>
+        /// <param name="credential">The token credential used to authenticate requests to the configuration store.</param>
+        /// <param name="optional">Determines the behavior of the App Configuration provider when an exception occurs while loading data from server. If false, the exception is thrown. If true, the exception is suppressed and no settings are populated from Azure App Configuration.
+        /// <exception cref="ArgumentException"/> will always be thrown when the caller gives an invalid input configuration (connection strings, endpoints, key/label filters...etc).
+        /// </param>
+        /// <returns>The provided configuration builder.</returns>
+        public static IConfigurationBuilder AddAzureAppConfiguration(
```

Comment:
This overload should be left out. We moved away from manually specifying replica endpoints in favor of auto discovery.